src/main/java/ru/l1ttleO/boyfriend/DelayedRunnable.java

```diff
@@ -0,0 +1,33 @@
+package ru.l1ttleO.boyfriend;
+
+import java.util.function.Consumer;
+import javax.annotation.Nullable;
+import org.jetbrains.annotations.NotNull;
+
+public class DelayedRunnable {
+
+    final public Thread thread;
+    final public long duration;
+    final public long startedAt;
+
+    public DelayedRunnable(final @Nullable ThreadGroup threadGroup, final @NotNull Consumer<DelayedRunnable> runnable, final @Nullable String name, final @NotNull long millis, final @Nullable Consumer<DelayedRunnable> ifInterrupted) {
```

Comment:
```suggestion
    public DelayedRunnable(@Nullable final ThreadGroup threadGroup, final @NotNull Consumer<DelayedRunnable> runnable, @Nullable final String name, final long millis, @Nullable final Consumer<DelayedRunnable> ifInterrupted) {
```
Reason: `Missorted modifiers`, `Primitive type members cannot be annotated`